src/docs/Contributing/Designers/NamingConvention/index.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: 'Naming convention'
+order: 5
+---
+
+> Mozaic is a product used by a lot of people. When so many people work together from different places on such a big project, you need to establish some ground rules that everyone needs to follow. These rules must be followed across every tool your team is using to create consistency and simplify the collaboration process.
+
+We chose to use english in all our files to be understandable by the most. We also chose to use the PascalCase standard in our prototyping tools.
+
+## File Naming Conventions
+
+We use two letters prefixes to name our files, this two letters depending on the pattern type and look like that: **_[XX] File name.sketch_**
+
+| Prefix   | Pattern type        | Example                                     |
+| -------- | ------------------- | ------------------------------------------- |
+| **[AA]** | Styles              | Colors, icons, fonts, shadows, ...          |
+| **[BA]** | Simple Components   | Buttons, links, tags, forms, ...            |
+| **[BB]** | Complex Components  | Notifications, accordions, popins, tabs ... |
+| **[ZZ]** | Specific librairies | Tools, documentation, ...                   |
+
+## Sketch
+
+Please follow these rules to have the same architecture and standards in every library. This will be very helpful for the design teams to understand every file in Mozaic.
```

Comment:
~lease follow these rules to have the same architecture and standards in every library. This will be very helpful for the design teams to understand every file in Mozaic.~ 
You need to follow these rules so we can assure the same architecture and standards in every library. This will avoid rework as we will review if each pattern submitted follow these standards.